### PR TITLE
[cppcheck] Change arguments of cppcheck

### PR DESCRIPTION
**Pull request by dongju-chae**
This ptach changes arguments of cppcheck.
`--std=posix` was deprecated.

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

### CODE REVIEW

File: ci/taos/plugins-good/pr-prebuild-cppcheck.sh [hunk 1]
```diff
@@ -72,19 +72,20 @@ function pr-prebuild-cppcheck(){
             case $i in
                 # in case of C/C++ code
                 *.c|*.cpp)
+                    # --std=posix is deprecated and removed in 2.05
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-prebuild-cppcheck.sh:75]: Looks good. let's merge this PR. 
It seems that we need to add an exception handling to handle various versions of CPPCheck software.

Comment (dongju-chae) [reply ci/taos/plugins-good/pr-prebuild-cppcheck.sh:75]: FYI, I uploaded `cppcheck` (2.2.0)  to nnstreamer ppa

**Review by leemgs — CHANGES_REQUESTED** [review]
Need to be updated for the maintenance of this module (CPPCheck).

File: ci/taos/plugins-good/pr-prebuild-cppcheck.sh (hunk 1 above)
Comment (dongju-chae) [reply ci/taos/plugins-good/pr-prebuild-cppcheck.sh]: I updated PR. Please review it again.

File: ci/taos/plugins-good/pr-prebuild-cppcheck.sh [hunk 2]
```diff
@@ -47,12 +51,20 @@ function pr-prebuild-cppcheck(){
...
     # Display the cppcheck version that is installed in the CI server.
     # Note that the out-of-date version can generate an incorrect result.
-    cppcheck --version
+    cppcheck_ver=$(cppcheck --version | awk {'print $2'})
+    echo $cppcheck_ver
+
+    default_cmd="--std=posix"
+    # --std=posix is deprecated and removed in 2.0.5
+    if [[ $(version $cppcheck_ver) -ge $(version "2.0.5") ]]; then
+        default_cmd="--library=posix"
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-prebuild-cppcheck.sh:66]: Looks good. 🥇

**Review by leemgs — APPROVED** [review]

**Pull request merged**